felix/fv/bpf_test.go

```diff
@@ -3842,3 +3846,46 @@ func checkServiceRoute(felix *infrastructure.Felix, ip string) bool {
 
 	return false
 }
+
+func checkIfPolicyProgrammed(felix *infrastructure.Felix, iface, hook, polName, action string) bool {
```

Comment:
imo should have bpf in its name. This function is then available for all FVs and can/should be used only in bpf mode.

Doesn't it duplicate what calico-bpf already does? Perhaps we can either grep calico-bpf output or we can have this as a lib function that calico-bpf can also reuse at least some parts of it, no?

Reply:
I think this will be a lot easier to iterate and find a match, rather using ```calico-bpf``` output as we need to do a bunch of string operations.